R/conjunct_test_linter.R

```diff
@@ -1,4 +1,4 @@
-#' Force && conditions in expect_true(), expect_false() to be written separately
+#' Force `&&` conditions in `expect_true()`, `expect_false()` to be written separately
```

Comment:
maybe `expect_true() and expect_false()` (i.e., no `,`)?

Reply:
Definitely. Changed to the suggested phrasing.